soccer/Processor.cpp

```diff
@@ -296,10 +299,19 @@ void Processor::run() {
             log->CopyFrom(packet->wrapper);
 
             curStatus.lastVisionTime = packet->receivedTime;
+
+            if (packet->wrapper.has_geometry()) {
+                // DEMO: Test out field sizes
+                const SSL_GeometryFieldSize fieldSize =
+                    packet->wrapper.geometry().field();
+                // FIXME - Account for network latency
```

Comment:
This was there before I started, I think i moved it up here by mistake. It was [here](https://github.com/RoboJackets/robocup-software/pull/544/files#diff-fefaade3f2ed6066fb659cd4ff181154L302). I'm not sure if it's relevant there, but I'll put it back if it is.